server/routes/api.js

```diff
@@ -1,6 +1,10 @@
 var express = require('express');
 var router = express.Router();
```

Comment:
@MrDwina the conflict is not resolved properly (there shouldn't be <<< >>>>

Reply:
where is this?